app/Http/Controllers/Admin/AdminController.php

```diff
@@ -15,21 +39,143 @@ class AdminController extends Controller
+                ->paginate(\App\User::PAGINATE_USER);
+            return view(
+                'admin.user.list', [
+                'users' => $users, 'roleAdmin' => $roleAdmin
+                ]
+            );
+        } else {
+            return redirect()->back();
+        }
     }
 
+    /**
+     * Log account admin out of the application.
+     *
+     * @return void
+     */
     public function userCreate()
     {
-        return view('admin.user.createOrUpdate');
+        $roleAdmin = \App\User::ADMIN;
+        return view('admin.user.createOrUpdate', ['roleAdmin' => $roleAdmin]);
+    }
+
+    /**
+     * Log account admin out of the application.
+     *
+     * @param Request $request Request create user
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function handleUserCreate(Request $request)
+    {
+        $this->validate(
+            $request, [
+            'username' => 'required',
+            'email' => 'required|unique:users,email',
+            'password' => 'required',
+            ]
+        );
+
+        $user = new User();
+        $user->name = $request->username;
+        $user->email = $request->email;
+        $user->password = bcrypt($request->password);
+        $user->can_see = \App\User::IS_TRUE;
+        $user->can_delete = \App\User::IS_TRUE;
+        if ($request->can_see == null) {
+            $user->can_see = \App\User::IS_FALSE;
+        }
+        if ($request->can_delete == null) {
+            $user->can_delete = \App\User::IS_FALSE;
+        }
+        $user->save();
+        return redirect()->back()->with('message', 'Create user success !');
+    }
+
+    /**
+     * Log account admin out of the application.
+     *
+     * @param $id Id user
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function userEdit($id)
+    {
+        $edit = User::find($id);
+        $roleAdmin = \App\User::ADMIN;
+        return view(
+            'admin.user.createOrUpdate', [
+            'edit' => $edit, 'roleAdmin' => $roleAdmin
+            ]
+        );
+    }
+
+    /**
+     * Log account admin out of the application.
+     *
+     * @param Request $request Request edit user
+     * @param $id      Id user
+     * 
+     * @return \Illuminate\Http\Response
+     */
+    public function handleUserEdit(Request $request, $id)
+    {
+        $user = User::find($id);
+
+        $user->can_see = \App\User::IS_TRUE;
+        $user->can_delete = \App\User::IS_TRUE;
+        if ($request->can_see == null) {
+            $user->can_see = \App\User::IS_FALSE;
+        }
+        if ($request->can_delete == null) {
+            $user->can_delete = \App\User::IS_FALSE;
+        }
+        $user->save();
+        return redirect()->back()->with('message', 'Edit user success !');
+    }
+
+    /**
+     * Log account admin out of the application.
+     *
+     * @param $id Id user
+     *
+     * @return \Illuminate\Http\Response
+     */
```

Comment:
incorrect whole comment . Check and Fix for all PR

Reply:
Fixed